proto/store/setting.proto

```diff
@@ -33,6 +33,27 @@ message WorkspaceProfileSetting {
 
   // The duration for refresh token.
   google.protobuf.Duration refresh_token_duration = 6;
+
+  // The setting of custom announcement
+  Announcement announcement = 7;
+}
+
+message Announcement {
+  // We support three levels of AlertLevel: INFO, WARNING, and ERROR.
+  enum AlertLevel {
+    ALERTLEVEL_INFO = 0;
+    ALERTLEVEL_WARNING = 1;
+    ALERTLEVEL_ERROR = 2;
```

Comment:
ERROR -> CRITICAL

We happen to use the same error color in the frontend. Semantically, this is CRITICAL instead of Error